src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
I confirm this corresponds to the real "dynamic" workflow. However, what may be a bit confusing here is that, if job.carryOn() is not active, we actually simulate the "static" workflow, for which there is no such timer. For the static workflow we wait for all jobs to finish before launching the merge (except for a very special case that we haven't yet included in the simulator). But I agree that the timer should not be completely removed since we will need it when job.carryOn() will be activated. 
To avoid confusion, we could add a few comments explaining (i) what version is being simulated (dynamic VS static) and (ii) what should be (de)activated when we switch versions. Or even better, we should implement the 2 cases distinctively.